src/NMS.AMQP/Provider/Amqp/AmqpConnection.cs

```diff
@@ -147,9 +147,9 @@ internal void OnRemoteOpened(Open open)
                     }
 
                     if (Array.Exists(capabilities,
-                        symbol => Equals(symbol, SymbolUtil.OPEN_CAPABILITY_DELAYED_DELIVERY)))
```

Comment:
where is delayed delivery gone?

Reply:
please have a look at preceeding if

Reply:
Ah ok right so this is just git diff playing about